src/app/loadout-drawer/loadout-drawer-reducer.ts

```diff
@@ -148,9 +152,7 @@ export function addItem(
     loadoutItem.equip = equip !== undefined ? equip : item.equipment && typeInventory.length === 0;
     // Reset all other items of this type to not be equipped
     if (loadoutItem.equip) {
-      for (const otherItem of typeInventory) {
-        otherItem.equip = false;
-      }
+      unequipOtherItems(defs, item, draftLoadout);
```

Comment:
This causes us to gain the behavior that you can no longer add two exotics in the same category to a loadout - if you do, one of them will unequip. This seems clearly correct, but can definitely lead to some briefly surprising behavior e.g. when using "fill in using equipped".